pkg/source/source.go

```diff
@@ -80,7 +81,12 @@ func (ks *Kind) Start(handler handler.EventHandler, queue workqueue.RateLimiting
 	// Lookup the Informer from the Cache and add an EventHandler which populates the Queue
 	i, err := ks.cache.GetInformer(ks.Type)
 	if err != nil {
-		return err
+		switch err.(type) {
+		case *meta.NoKindMatchError:
+			return fmt.Errorf("must install CRD %v before calling Start", err.(*meta.NoKindMatchError).GroupKind)
```

Comment:
maybe use `%s` here instead to get it pretty-printed?

Reply:
Updated.
They are the same output.